sklearn/neighbors/_ball_tree.pyx

```diff
@@ -5,15 +5,15 @@ __all__ = ['BallTree']
 
 DOC_DICT = {'BinaryTree': 'BallTree', 'binary_tree': 'ball_tree'}
 
-VALID_METRICS = ['EuclideanDistance', 'SEuclideanDistance',
-                 'ManhattanDistance', 'ChebyshevDistance',
-                 'MinkowskiDistance', 'WMinkowskiDistance',
-                 'MahalanobisDistance', 'HammingDistance',
-                 'CanberraDistance', 'BrayCurtisDistance',
-                 'JaccardDistance', 'DiceDistance',
-                 'RogersTanimotoDistance', 'RussellRaoDistance',
-                 'SokalMichenerDistance', 'SokalSneathDistance',
-                 'PyFuncDistance', 'HaversineDistance']
+VALID_METRICS = ['EuclideanDistance64', 'SEuclideanDistance64',
```

Comment:
Is it possible to reformat this list so that's it is alphabetically ordered with a element per line?